js/animateCSS.js

```diff
@@ -0,0 +1,165 @@
+/* MagicMirror²
+ * AnimateCSS System from https://animate.style/
+ * by @bugsounet
+ * for Michael Teeuw https://michaelteeuw.nl
+ * MIT Licensed.
+ */
+
+/* enumeration of animations in Array **/
+const _AnimateCSSIn = [
```

Comment:
that _ in front of the name is a Pattern we dont use anywhere else. what should it signal?